SharedPackages/BrowserServicesKit/Sources/Subscription/Managers/SubscriptionManagerV2.swift

```diff
@@ -430,27 +465,13 @@ public final class DefaultSubscriptionManagerV2: SubscriptionManagerV2 {
         Logger.subscription.debug("Get tokens \(policy.description, privacy: .public)")
 
         do {
-            let currentCachedTokenContainer = try oAuthClient.currentTokenContainer()
-            let currentCachedEntitlements = currentCachedTokenContainer?.decodedAccessToken.subscriptionEntitlements
-
             await migrateAuthV1toAuthV2IfNeeded()
 
             let resultTokenContainer = try await oAuthClient.getTokens(policy: policy)
             let newEntitlements = resultTokenContainer.decodedAccessToken.subscriptionEntitlements
             cachedUserEntitlements = newEntitlements
```

Comment:
Just double checking here... is this the right place to be updating this?  It may be - just wondering if this is the entry point for the entitlements changing or if these should've been caught in the other places we already have.